draft-dnsop-deleg-transport.md

```diff
@@ -208,31 +217,33 @@ alpn=doq tlsa=disabled mandatory=alpn
 ~~~
 {: title="tlsa SvcParam Examples for DELEG"}
 
-## Deployment Considerations
+## Resolution procedure
 
-The DELEG and SVCB records intends to replace the NS record while also adding additional functionality in order to support additional transports for the DNS. Below are discussions of considerations for deployment.
+As specified in {{?I-D.draft-dnsop-deleg}}, the ServiceMode DELEG records provide a weak signal from the operator about which transport mechanisms they would most prefer clients to use, but it is not mandatory that resolvers try the delegated servers in priority order.  It might not even be possible for a resolver to do, such as if the highest priority record specifies an alpn that the resolver does not implement, but a lower priority record has an alpn that it can use.
```

Comment:
I don't understand this line.  Each ServiceMode record documents the supported transports for a particular endpoint, and the SvcPriority documents the recommended trial order.  Incompatible endpoints are skipped.  Of course clients are free to reorder them based on additional heuristics.  This is all covered in RFC 9460.

Reply:
What don't you understand then?  It's just making it clear in this document as well.   Do you think this statement is inconsistent with that, or should just be shorter in drawing attention to it?

Reply:
OK.  If this is just meant to be highlighting some relevant aspects of RFC 9460, then it should say so.

Reply:
Will do.  I'll be working on some more editing tomorrow and will add the explicit reference.